test/lib/import/brca/providers/royal_marsden/royal_marsden_handler_test.rb

```diff
@@ -45,6 +45,54 @@ def setup
     normal_record.raw_fields['teststatus'] = 'NO PATHOGENIC VARIANT IDENTIFIED'
     @handler.process_teststatus(@genotype, normal_record)
     assert_equal 1, @genotype.attribute_map['teststatus']
+
+    non_path_record = build_raw_record('pseudo_id1' => 'bob')
+    non_path_record.raw_fields['teststatus'] = 'Ex del'
```

Comment:
Can `raw_fields` be passed to the `build_raw_record` method, eg:

`broken_record = build_raw_record('pseudo_id1' => 'bob', raw_fields: { 'teststatus' => 'Ex del' } )`

Reply:
Thanks for the suggestion  @ollietulloch ! I have tried that and it doesn't work-  I think its because build new record adds the test status field by loading the rawtext_clinical_json. So that needs to be loaded first then we can update the test status afterwards. Let me know if there is another way!